website/content/docs/internals/security.mdx

```diff
@@ -307,21 +293,21 @@ The following are not part of the threat model for client agents:
 
 There are two main components to consider to for external threats in a Nomad cluster:
 
-- **Server agent** - Internal cluster leader elections and replication is
+- **Server agent** - Internal cluster leader elections, and replication is
   managed via Raft between server agents encrypted in transit. However,
   information about the server is stored unencrypted at rest in the agent's
-  data directory. This may contain sensitive information such as ACL tokens
+  data directory. This may contain sensitive information such as ACL tokens,
   and TLS certificates.
 
 - **Client agent** - Client-to-server communication within a cluster is
-  encrypted and authenticated using mTLS. Information about the allocations on
-  a client node is unencrypted in the agent's data and configuration
+  encrypted, and authenticated using mTLS. Information about the allocations on
+  a client node is unencrypted in the agent's data, and configuration
   directory.
```

Comment:
This bit got "wat?" really fast...not because of your edits. I'm not 100% sure of the intention here. Perhaps it's to mention that the data directory and configuration are both stored to disk in the clear.

I think this was kind of the point, but it sort of fell apart at the end for me. I'd love to hear your thoughts on it, @picatz .

- Client state contains information about the allocations running on a client.
- The allocation directory, generally a child directory of data_dir but not required to be, can contain logs and files created or read during an allocations lifetime.
- The configuration directory could have elements that should be protected.

Reply:
The intention was to highlight that information is stored in the clear on disk on client nodes, but information is encrypted on-the-wire. I think the wording could def be adjusted, just not exactly sure what that wording is yet. 🤔

Maybe:
>  Client state that contains all information of running allocations is stored in the clear on disk on client nodes. This information is encrypted over-the-wire when communicating with server nodes.

What do you think @angrycub? Maybe it needs to be a bit more verbose?